mycore-viewer/src/main/java/org/mycore/viewer/configuration/MCRViewerIIIFConfiguration.java

```diff
@@ -33,10 +33,10 @@ public MCRViewerIIIFConfiguration setup(HttpServletRequest request) {
         final String derivate = getDerivate(request);
 
         setProperty("manifestURL", MCRFrontendUtil.getBaseURL()
-            + MCRConfiguration2.getString("MCR.Viewer.IIIF.URL.Presentation").orElse("api/iiif/v2/presentation/mets/")
+            + MCRConfiguration2.getString("MCR.Viewer.IIIF.URL.Presentation").orElse("api/iiif/presentation/v2/mets/")
             + derivate + "/manifest");
         setProperty("imageAPIURL", MCRFrontendUtil.getBaseURL()
-            + MCRConfiguration2.getString("MCR.Viewer.IIIF.URL.Image").orElse("api/iiif/v2/image/Iview/"));
+            + MCRConfiguration2.getString("MCR.Viewer.IIIF.URL.Image").orElse("api/iiif/image/v2/"));
```

Comment:
no orElse() as we have default values defined in mycore.properties, use getStringOrThrow() instead